src/main/resources/templates/smallfull/criteria.html

```diff
@@ -4,86 +4,107 @@
-
-            <fieldset class="form-group" th:classappend="${#fields.hasErrors('isCriteriaMet')} ? 'error' : ''">
-                <legend class="visuallyhidden">Full accounts criteria radio buttons"</legend>
-                <div class="form-group">
-                    <span id="isCriteriaMet-errorId"
-                          th:if="${#fields.hasErrors('isCriteriaMet')}"
-                          th:each="e : ${#fields.errors('isCriteriaMet')}" th:text="${e}" class="error-message">
-                    </span>
-
-                    <label class="block-label selection-button-radio" for="yesButton" id="yesButton-label">
-                        <input id="yesButton"
-                               type="radio"
-                               value="yes"
-                               th:field="*{isCriteriaMet}"
-                               th:errorclass="error-message"
-                               class="piwik-event" data-event-id="Yes"/>Yes
-                    </label>
-
-                    <label class="block-label selection-button-radio" for="noAlternativeFilingButton" id="noAlternativeFilingButton-label">
-                        <input id="noAlternativeFilingButton"
-                               type="radio"
-                               value="noAlternativeFilingMethod"
-                               th:field="*{isCriteriaMet}"
-                               th:errorclass="error-message"
-                               class="piwik-event" data-event-id="No other full"/>No - show me other ways to file full accounts
-                    </label>
-
-                    <label class="block-label selection-button-radio" for="noOtherAccountsButton" id="noButton-label">
-                        <input id="noOtherAccountsButton"
-                               type="radio"
-                               value="noOtherAccounts"
-                               th:field="*{isCriteriaMet}"
-                               th:errorclass="error-message"
-                               class="piwik-event" data-event-id="No other accounts"/>No - I'll file other types of accounts
-                    </label>
-                </div>
-            </fieldset>
-
-            <div class="form-group">
-              <input id="next-button" class="button" type="submit" role="button" value="Continue"/>
+<head>
+  <title>Full Accounts Criteria</title>
+</head>
+<body>
+<div id="criteria-main-content" layout:fragment="content">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form id="criteria-form" th:action="@{''}" th:object="${criteria}" class="form" method="post">
+        <div th:replace="fragments/globalErrors :: globalErrors"></div>
+        <header>
+          <h1 id="page-title" class="govuk-heading-l">
+            File full accounts
+          </h1>
+        </header>
+
+        <h2 class="govuk-heading-m" id="criteria-sub-heading">Criteria</h2>
+
+        <div class="panel panel-border-wide">
```

Comment:
I followed the prototype to all my changes so that they both were in line. I've basically copy and paste from the prototype thinking that was up-to-date and using the correct classes. 
I will remove them to see if they keep the style. If they don't apply any styling, I will remove them.